src/bun.js/builtins/cpp/BundlerPluginBuiltins.cpp

```diff
@@ -48,7 +48,7 @@ namespace WebCore {
 const JSC::ConstructAbility s_bundlerPluginRunOnResolvePluginsCodeConstructAbility = JSC::ConstructAbility::CannotConstruct;
 const JSC::ConstructorKind s_bundlerPluginRunOnResolvePluginsCodeConstructorKind = JSC::ConstructorKind::None;
 const JSC::ImplementationVisibility s_bundlerPluginRunOnResolvePluginsCodeImplementationVisibility = JSC::ImplementationVisibility::Public;
-const int s_bundlerPluginRunOnResolvePluginsCodeLength = 3593;
+const int s_bundlerPluginRunOnResolvePluginsCodeLength = 3619;
```

Comment:
This and other changes in this file were auto-generated. I suppose that's expected?

Reply:
Yeah this was fixed on main before you rebased last